Sofascore2025/EventView.swift

```diff
@@ -0,0 +1,290 @@
+import UIKit
+import SnapKit
+import SofaAcademic
+
+
+class EventView: BaseView {
+	private var _match : Event
+	private var homeTeamImage = UIImageView()
+	private var awayTeamImage = UIImageView()
+	private var homeTeamName = UILabel()
+	private var awayTeamName = UILabel()
+	private var homeTeamScore = UILabel()
+	private var awayTeamScore = UILabel()
+	private var startTime = UILabel()
+	private var clock = UILabel()
+	private var splitLabel = UIImageView()
+	
+	init(match: Event) {
+		self._match = match
+		super.init()
+	}
```

Comment:
moze prazan red uvijek nakon metode
```suggestion
	}
	
```